### DISCUSSION TITLE
課題12についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#46] [Q&A]
問題１．２．５は作成できたのですが、３と４はとても難しいです。
３はあと少しでわかりそうな気もするのですが、4に関しては2つ目のfor文の作り方が特にわかりませんでした。

**Comment by nsc-kiwamu** [comment]
作成したものを一旦、pushして下さい。
ものが無いとコメントのしようも無いので。。。

**Reply by TakedaAyano** [reply]
すみません、pushをし忘れておりました。
只今、pushいたしました。
